.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
```

Comment:
Purely personal opinion: I'm not a big fan of using `.github` directory for files that aren't GitHub-specific. In this PR, `CONTRIBUTING.md` and `CODE_OF_CONDUCT.md` aren't, while `ISSUE_TEMPLATE.md` is. Put another way, the the first two should be read by someone just browsing, while the last is of no use unless one is filing an issue.